main-api/controllers/hg_controller.py

```diff
@@ -0,0 +1,47 @@
+from typing import List, Tuple, Union
+
+import connexion
+from connexion import problem
+from connexion.lifecycle import ConnexionResponse
+from flask import current_app
+from gpa_api.models.hg_fluid import HgFluid
+from services import FluidService
+
+
+def create_fluid() -> Union[Tuple[HgFluid, int], ConnexionResponse]:
+    """Create hg fluid
+
+    :rtype: Fluid
+    """
+    if connexion.request.is_json:
+        fluid = HgFluid.from_dict(connexion.request.get_json())
+        fluid_service = FluidService(current_app.db_session)
+        return fluid_service.create(fluid), 201
```

Comment:
What happens if fluid_service.create fails? should the response then be of another status code/type

Reply:
Uncaught errors results in a default 500 error handled by flask/connexion

Reply:
great